include/drivers/spi.h

```diff
@@ -114,6 +114,16 @@ extern "C" {
  */
 #define SPI_CS_ACTIVE_HIGH	BIT(14)
 
+/**
+ * @brief SPI Frame Format
+ *
+ * 2 frame formats are exposed: TI and Motorola.
+ * By default, and because this is the one which is mostly used, the TI frame
+ * frame format will prevail.
```

Comment:
Please add a link to somewhere where the formats are specified.

Reply:
Many sources telling it's only a matter of CPOL/CPHA, but it seems to be a little bit more than that (first bit tx/rx timing). If it was only cpol/cpha it would be completely useless to expose such choice on hardware level. It's clearer in datasheets.

Just realized that I mixed the order, Motorola is the default... Will change that.

Reply:
I'll clarify the documentation directly. No links seems to be worth